components/centraldashboard/public/.eslintrc.json

```diff
@@ -6,6 +6,7 @@
     "extends": ["eslint:recommended", "google"],
     "globals": {
         "VERSION": true,
```

Comment:
Shouldn't version be a `string` perhaps `"dev-build"`?

Reply:
Actually, this is just indicating to ESLint that the value is considered to be in-scope at runtime so the linter doesn't complain.

Based on the [docs](https://eslint.org/docs/user-guide/configuring#specifying-globals), it looks like the best option is to set them to **readonly**.

Reply:
I see, thanks!